### PR TITLE
tcps db conn fix : disabled oob in sqlnet.ora of DB wallet

### PR DESCRIPTION
**Pull request by vatsrahul**
Added Disable_oob=on in sqlnet.ora of db wallet in configTcps.sh
For version 19 and 21c.

Tested both tcp and tcps connection thereafter.

### CODE REVIEW

**Review by yunus-qureshi — APPROVED** [review]

File: OracleDatabase/SingleInstance/README.md [hunk 1]
```diff
@@ -239,17 +239,17 @@ To configure  wallet password, please use the following command:
...
+* To easily connect with the database via TCPS users can use EZCONNECT (easy connect naming method) as shown:
 
-        # Connecting Enterprise Edition
-        sqlplus sys@ORCLCDB as sysdba
-        # Connecting Express Edition
-        sqlplus sys@XE as sysdba
+        sqlplus sys@tcps://<host>:<port>/<service_name>?wallet_location=<wallet_directory> as sysdba
+        # service_name could be ORACLE_SID or ORACLE_PDB
+        # port is mapped port of host where container is running
+        # wallet_directory is where client walllet is copied to.
```
Comment (yunus-qureshi) [inline OracleDatabase/SingleInstance/README.md:248]: walllet typo

File: OracleDatabase/SingleInstance/README.md [hunk 2]
```diff
@@ -239,17 +239,17 @@ To configure  wallet password, please use the following command:
...
-* After `tnsnames.ora` is modified, go inside the downloaded client wallet directory and set TNS_ADMIN for SQL\*Plus by using the `export TNS_ADMIN=$(pwd)` command. Then users can connect via TCPS with, for example, the following commands:
+* The client wallet directory above will include wallet files, along with sample `sqlnet.ora` and `tnsnames.ora` files.
+* To connect to the database via TCPS use EZCONNECT (easy connect naming method) as shown:
 
-        # Connecting Enterprise Edition
-        sqlplus sys@ORCLCDB as sysdba
-        # Connecting Express Edition
-        sqlplus sys@XE as sysdba
+        sqlplus sys@tcps://<host>:<port>/<service_name>?wallet_location=<wallet_directory> as sysdba
+        # service_name could be ORACLE_SID or ORACLE_PDB
```
Comment (cjbj) [inline OracleDatabase/SingleInstance/README.md:246]: (i) by implicit definition service_name is a Service Name, not a SID
(ii) SIDs don't work with Easy Connect syntax used so SID should be removed from the statement

What is ORACLE_PDB?  This is a non (traditional) standard variable.

**Review by cjbj — CHANGES_REQUESTED** [review]

File: OracleDatabase/SingleInstance/README.md [hunk 3]
```diff
@@ -239,17 +239,17 @@ To configure  wallet password, please use the following command:
         # ORACLE_SID default value is ORCLCDB
         docker cp <container name>:/opt/oracle/oradata/clientWallet/<ORACLE_SID> <destination directory>
 
-* The client wallet directory above will include wallet files, along with sample `sqlnet.ora` and `tnsnames.ora` files. You should edit the `HOST` and `PORT` fields accordingly in the `tnsnames.ora` before connecting using TCPS.
-* After `tnsnames.ora` is modified, go inside the downloaded client wallet directory and set TNS_ADMIN for SQL\*Plus by using the `export TNS_ADMIN=$(pwd)` command. Then users can connect via TCPS with, for example, the following commands:
+* The client wallet directory above will include wallet files, along with sample `sqlnet.ora` and `tnsnames.ora` files.
+* To connect to the database via TCPS use EZCONNECT (easy connect naming method) as shown:
```
Comment (cjbj) [inline OracleDatabase/SingleInstance/README.md:243]: Use the formal anme 'Easy Connect', not the old, mostly internal name EZCONNECT.  Also users are technically not restricted to using Easy Connect.  And there's arguably no need to mention it at all??  So the sentence can be like:
```
To connect to the database via TCPS you can use SQL*Plus like:
```

File: OracleDatabase/SingleInstance/README.md (hunk 2 above)
Comment (vatsrahul) [reply OracleDatabase/SingleInstance/README.md]: removed that comment

**Review by yunus-qureshi — APPROVED** [review]

**Review by yunus-qureshi — APPROVED** [review]

**Comment by vatsrahul** [comment]
@gvenzl Can you merge this PR ?

**Review by cjbj — APPROVED** [review]
I have no further comments about the area that previously concerned me

**Review by gvenzl — APPROVED** [review]

**Pull request merged**